Source/Plugins/EditorModules/ObjectInspector/ObjectInspectorPlugin.cs

```diff
@@ -30,7 +24,7 @@ internal static ObjectInspectorPlugin Instance
 
 		private	List<ObjectInspector>	objViews		= new List<ObjectInspector>();
 		private	bool					isLoading		= false;
-
+		private ObjectInspectorSettings userSettings;
```

Comment:
Local style: Looks like the fields are meant to be laid out table-style, but the usage of inline tabs for alignment messes it up. Can you do a drive-by fix here and either remove the alignment from all the fields, or fix it by using inline-spaces? If you do the latter, make sure to also include the new field.